po/de.po

```diff
@@ -1216,12 +1275,31 @@ msgid ""
 "user manual, found through the &quot;Help&quot; menu item in the main "
 "window's menu button."
 msgstr ""
+"GTG kann Aufgaben auch nach ihrer Beschreibung suchen, so dass es einfach "
+"ist, nach spezifische Aufgaben zu suchen.\n"
+"\n"
+"Die Suche ist einfach: Drücken Sie Strg+F und schreibe dass, was Sie suchen "
+"wollen, in die Suche.\n"
+"\n"
+"GTG kann Suchvorgänge in der Seitenleiste unter &quot;Gespeicherte "
+"Suchvorgänge&quot; speichern. Damit ist es möglich, vorherige Suchen wieder "
+"durchzuführen. Die Suchergebnisse werden automatisch aktualisiert.\n"
+"\n"
+"Die Suche in GTG ist sehr mächtig, es akzeptiert viele Parameter um nach "
+"bestimmte Aufgaben zu suchen. Zum Beispiel, mit der Suche &quot;@Auftrag !"
+"Heute&quot; sucht man nach Aufgaben, die das Schlagwort @Auftrag besitzen "
```

Comment:
"Zum Beispiel sucht man mit der Suche &quot;@​Auftrag !Heute&quot; nach Aufgaben…" instead of "Zum Beispiel, mit der Suche &quot;@​Auftrag !Heute&quot; sucht man nach Aufgaben…" sounds much better.